modules/programs/rbw.nix

```
@@ -0,0 +1,95 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.programs.rbw;
+
+  jsonFormat = pkgs.formats.json { };
+
+  settingsModule = with lib;
+    types.submodule {
+      options = {
+        email = mkOption {
+          type = types.str;
+          example = "name@example.com";
+          description = "The email address for your bitwarden account.";
+        };
+
+        base_url = mkOption {
+          type = types.str;
+          default = null;
+          example = "bitwarden.example.com";
+          description =
+            "The base-url for a self-hosted bitwarden installation.";
+        };
+
+        identity_url = mkOption {
+          type = types.str;
+          default = null;
+          example = "identity.example.com";
+          description = "The identity url for your bitwarden installation.";
+        };
+
+        lock_timeout = mkOption {
+          type = types.ints.unsigned;
+          default = 3600;
+          example = 300;
+          description = ''
+            The amount of time that your login information should be cached.
+          '';
+        };
+
+        pinentry = mkOption {
+          type = types.package;
+          default = pkgs.pinentry;
+          defaultText = literalExample "pkgs.pinentry";
+          example = literalExample "pkgs.pinentry-gtk2";
+          description = ''
+            The pinentry program to use for password prompts.
+          '';
+        };
+      };
+    };
+in {
+  meta.maintainers = with lib.hm.maintainers; [ ambroisie ];
+
+  options.programs.rbw = with lib; {
+    enable = mkEnableOption "rwb, a CLI Bitwarden client";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.rbw;
+      defaultText = literalExample "pkgs.rbw";
+      description = ''
+        Package providing the <command>rbw</command> tool and its
+        <command>rbw-agent</command> daemon.
+      '';
+    };
+
+    settings = mkOption {
+      type = types.nullOr settingsModule;
+      default = null;
+      defaultText = literalExample ''
+        {
+          email = "name@example.com";
+          lock_timeout = 300;
+          pinentry = pkgs.pinentry-gtk2;
+        }
+      '';
+      description = ''
+        <command>rbw</command> configuration, if not defined the
+        configuration will not be managed by <command>home-manager</command>.
```

```suggestion
        rbw configuration, if not defined the
        configuration will not be managed by Home Manager.
```

Since you talking about projects, you don't need to add a `<command>` tag.